pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[build-system]
+requires = [
+  "setuptools>=51.0",
+  "wheel>=0.36",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
```

Comment:
@asvetlov it seems like you forgot to add `setuptools-scm[toml]` to `[build-system]requires`. Adding it to `setup_requires` is rather deprecated (and arguably doesn't work as expected).

Reply:
Thank you very much.
Can I drop `setuptools-scm` from `setup.cfg` entirely?
IIRC SCM doesn't work if `pyproject.toml` contains `setuptools-scm` requirements (without `[toml]` extra).

Reply:
Yes, you should add it with `[toml]` and then, it'll work. No need to have it in `setup.cfg`. `setup_requires` is something weird happening inside of setuptools, after a PEP 517 front-end (pip/build/whatever) has already prepared an isolated virtualenv for the project based on the settings in `pyproject.toml`. I used to keep it in `setup.py` only as for compatibility with the ancient pre-PEP517 toolchain.

Reply:
Ok. Cleaned up on the master.